packages/world/ts/config/v2/output.ts

```diff
@@ -1,4 +1,47 @@
 import { Config as StoreConfig, Table } from "@latticexyz/store/config/v2";
+import { ModuleConfig } from "./input";
+
+export type SystemConfig = {
+  /** The full resource selector consists of namespace and name */
```

Comment:
is this an old comment? not sure what it's referring to

Reply:
yeah was an old comment, updated